powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/PowerAuthClient.java

```diff
@@ -2135,4 +2135,79 @@ RecoveryCodeActivationResponse createActivationUsingRecoveryCode(String recovery
      */
     TelemetryReportResponse requestTelemetryReport(TelemetryReportRequest request, MultiValueMap<String, String> queryParams, MultiValueMap<String, String> httpHeaders) throws PowerAuthClientException;
 
+    /**
+     * Create an application configuration.
+     * @param request Create application configuration request.
+     * @return Create application configuration response.
+     * @throws PowerAuthClientException In case REST API call fails.
+     */
+    CreateApplicationConfigResponse createApplicationConfig(CreateApplicationConfigRequest request) throws PowerAuthClientException;
+
+    /**
+     * Create an application configuration.
+     * @param request Create application configuration request.
+     * @return Create application configuration response.
+     * @throws PowerAuthClientException In case REST API call fails.
+     */
+    CreateApplicationConfigResponse createApplicationConfig(CreateApplicationConfigRequest request, MultiValueMap<String, String> queryParams, MultiValueMap<String, String> httpHeaders) throws PowerAuthClientException;
+
+    /**
+     * Create an application configuration.
+     * @param applicationId Application identifier.
+     * @param key Configuration key.
+     * @param values Configuration values.
+     * @return Create application configuration response.
+     * @throws PowerAuthClientException In case REST API call fails.
+     */
+    CreateApplicationConfigResponse createApplicationConfig(String applicationId, String key, List<String> values) throws PowerAuthClientException;
```

Comment:
What about keeping only the method with `CreateApplicationConfigRequest` and not listing all arguments?

Reply:
I did this for consistency with other methods in the client, if there are few arguments, we provide such methods.